libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -58,32 +60,37 @@
   private final String accessToken;
   private final String userAgent;
   private final OkHttpClient client;
+  private final SharedPreferences sharedPreferences;
+  private List<String> revokedKeys;
```

Comment:
This variables is updated below within `addSHAtoKeys()` so that we are updating the keys immediately, so there isn't a 24hr delay on getting the latest revokedKeys to be used by the system. It should stay up-to-date and within a state of flux. This is why it was not made a final variable. Do you have recommendations for changing how this works?